glusterhealth/reports/disk_usage.py

```diff
@@ -0,0 +1,33 @@
+#
+# Copyright (c) 2017 Red Hat, Inc.
+#
+# This file is part of gluster-health-report project which is a
+# subproject of GlusterFS ( www.gluster.org)
+#
+# This file is licensed to you under your choice of the GNU Lesser
+# General Public License, version 3 or any later version (LGPLv3 or
+# later), or the GNU General Public License, version 2 (GPLv2), in all
+# cases as published by the Free Software Foundation.
+
+import logging
+from utils import get_disk_usage_details
+	
+def check_disk_usage_percentage(ctx, path, percentage=0):
+	out = get_disk_usage_details(path)
+	if out is None:
+		return
+	if out.percentage:
+		used_percent = int(out.percentage.split('%')[0])
+		if used_percent >= percentage:
+			ctx.notok("Disk used percentage for \'"+ path +"\' is exceeding "+ str(percentage) +"%, consider deleting unneccesary data")
```

Comment:
Please change this to

ctx.notok("Disk used percentage is exceeding, consider deleting unneccesary data", path=path, percentage=percentage)